docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -15,20 +15,20 @@ services apply the change right after custom resource (CR) is patched. Compute d
 
 .Procedure
 
-. Wait for cell1 Compute data plane services version updated (it may take some time):
+. Wait for cell1 Compute data plane services version to be updated (it may take some time):
```

Comment:
What version are users updating to? 

Also, there is a hackfest comment about this step. https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ24DLt4 

Can we give more guidance on how long it might take, and what to do if the update fails? Where can the customer look for issues?

Reply:
Maybe we could say something like "Wait for cell1 Compute data plane services version to be updated from RHOSP 17.1 to RHOSO 18.0. Minimal downtime is expected."
Then upstream only: "For more guidance about reducing downtime, see https://docs.openstack.org/nova/latest/admin/upgrades.html#rolling-upgrade-process".

Reply:
regarding downtime, there should be no downtime for running VMs

Reply:
@bogdando If there should not be any downtime for running VMs, then the following text doesn't make sense to include: "...Minimal downtime is expected. For more guidance about reducing downtime, see https://docs.openstack.org/nova/latest/admin/upgrades.html#rolling-upgrade-process".

Or is there another scenario (other than running VMs) where downtime is expected?

Reply:
no downtime for VM workloads expected